### PR TITLE
(feat) event manager and profile resolves #21

### LINKED ISSUE #21: (feat) manage events profile

### PR DESCRIPTION
**Pull request by CaptainKevin**

### CODE REVIEW

File: public/eventManager/eventManagerTemplate.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div>
+  <div>
+    <h3>Hosted Events:</h3>
+    <div ng-repeat="hostedEvent in event.hosted">
+      <h5>{{hostedEvent.title}}:</h5>
+      <ul>
+        <li ng-repeat="users in hostedEvent.usersApplied"><a ui-sref="userProfile({username: users.username})">{{users.username}}  </a><button ng-class="{true: 'btn btn-success disabled', false: 'btn btn-primary'}[users.confirmed]" ng-disabled="users.confirmed" ng-click="event.confirmUser(users.username); users.confirmed = !users.confirmed" >{{!users.confirmed && 'Accept' || 'Accepted'}}</button></li>
```
Comment (MichaelGofron) [inline public/eventManager/eventManagerTemplate.html:10]: Comment or break up this html code. It's very difficult to read and understand

File: public/app.js [hunk 2]
```diff
@@ -1,29 +1,61 @@
...
+        resolve: {
+          getProfilePrep: getProfileService
+        }
+      })
+  }
+
+
+
+  function getEventsService ($http) {
+    return $http({method: 'GET', url: '/user/eventsManager'});
```
Comment (MichaelGofron) [inline public/app.js]: Move $http to data factory

File: bower.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "angular": "~1.4.4",
     "angular-ui-router": "~0.2.15",
     "bootstrap": "~3.3.5",
-    "angular-bootstrap": "~0.13.3"
+    "angular-bootstrap": "~0.13.3",
```
Comment (MichaelGofron) [inline bower.json]: remove comma

File: public/services/confirmEvent.service.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+(function(){
+  angular
+      .module('app')
+      .factory('confirmEvent', confirmEvent);
+
+  confirmEvent.$inject = ['$http'];
+
+  function confirmEvent($http){
+    return {
```
Comment (MichaelGofron) [inline public/services/confirmEvent.service.js:9]: Move to data.service.js

**Comment by CaptainKevin** [comment]
8/29 refactor - ready for pull request review

**Pull request merged**